### PR TITLE
added domain_parameters to domain show.json.rabl

### PR DESCRIPTION
**Pull request by isratrade**

### CODE REVIEW

File: app/models/domain.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def to_param
   end
 
   def as_json(options={})
-    super({:only => [:name, :id]}.merge(options))
+    super.merge(options)
```
Comment (ohadlevy) [inline app/models/domain.rb:34]: shouldnt we then add :include => :domain_parameters here too?

Comment (ohadlevy) [inline app/models/domain.rb:34]: or maybe i mixing up the commits :) sorry, to many pull requests for the same code :)

Comment (isratrade) [inline app/models/domain.rb:34]: @ohadlevy , this added to the domains_controller by @jpoppe
https://github.com/theforeman/foreman/pull/200

**Comment by isratrade** [comment]
domain parameters can be in API v2

**Pull request closed without merging**